webdriver-spec.html

```diff
@@ -734,8 +734,9 @@
   to the description of <var>status</var> in the <a>status code registry</a>.
 
  <li><p>If <var>data</var> is not null, let <var>response</var>’s
-  <a>body</a> be the result of <a>serializing as JSON</a>
-  with <var>data</var> as the argument.
+  <a>body</a> be a JSON object with a top-level property
```

Comment:
I also think saying “top-level property” is redundant. You can just say `… JSON <a>Object</a> with a key <code>value</code> set to the <a>JSON serialisation</a> of <var>data</var>`.